addons/account_edi/models/account_edi_document.py

```diff
@@ -168,11 +168,22 @@ def _postprocess_cancel_edi_results(documents, edi_result):
                         'error': move_result.get('error', False),
                         'blocking_level': move_result.get('blocking_level', DEFAULT_BLOCKING_LEVEL) if move_result.get('error') else False,
                     })
+<<<<<<< HEAD
 
             if move_ids_to_cancel:
                 invoices = self.env['account.move'].browse(list(move_ids_to_cancel))
                 invoices.button_draft()
                 invoices.button_cancel()
+||||||| parent of 5fb4334888e (temp)
+
+            if moves_to_cancel:
+                moves_to_cancel.button_draft()
+                moves_to_cancel.button_cancel()
+=======
+            if moves_to_cancel and all((doc.state == 'cancelled' and doc.blocking_level != 'error') or not doc.edi_format_id._needs_web_services() for doc in moves_to_cancel.edi_document_ids):
```

Comment:
@jva-odoo It is actually even wrong from the beginning because we should do it move by move...

Reply:
@jco-odoo okay, but do we still continue this FW port or not?

Reply:
@jva-odoo Yes, but it needs to be fixed like in the original of https://github.com/odoo/odoo/pull/127862 and we need to do the same for the saas-15.2 one.